src/layout/layout.ts

```diff
@@ -68,7 +68,7 @@ class Layout implements IComponent {
   // them with real data asynchronously (if it takes too long to do at once) by yielding a thread
   // and returning to it later through a set timeout
   private discoverDom() {
-    let discoverTime = getPageContextBasedTimestamp();
+    let discoverTime = Math.round(getPageContextBasedTimestamp());
```

Comment:
My personal preference is to keep the function clean and doing exactly what it says. Also, doing so won't eliminate all Math.round() calls in the code base, because some timestamps come to us from, for example, performance APIs, so we still have to manually round them.